Frontend/ui-driver/src/Helpers/Utils.js

```diff
@@ -675,19 +675,30 @@ function rideLabelConfig(){
 export const getPastDays = function (count) {
   try {
     let result = [];
+    const language = JBridge.getFromSharedPrefs("LANGUAGE_KEY");
     for (var i = 0; i < count; i++) {
       let d = new Date();
       d.setDate(d.getDate() - i);
-      let obj = { utcDate: d.toISOString(), date: d.getDate(), month: d.toLocaleString('default', { month: 'short' }), year: d.getFullYear() };
+      let obj = { utcDate: d.toISOString(), date: d.getDate(), month: d.toLocaleString(getFormattedLanguage(language), { month: 'short' }), year: d.getFullYear() };
       result.push(obj);
     }
+    console.log(language, getFormattedLanguage(language))
     console.log(result);
     return result.reverse();
   } catch (e) {
     console.log("error in getPastDays", e);
   }
 };
 
+function getFormattedLanguage(language){
+  if (language == "EN_US") return "en-us";
+  else if (language == "HI_IN") return "hi-in";
+  else if (language == "KN_IN") return "kn-in";
+  else if (language == "TA_IN") return "ta-in";
+  else if (language == "BN_IN") return "bn-in";
+  else return "en-us";
+}
+
```

Comment:
we can use same function from common only

Reply:
done